tools/python_path.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import skbuild
+from skbuild import cmaker
+
+
+def main():
+    if len(sys.argv) < 2:
+        print("Usage: ", sys.argv[0], "bin|lib|include")
+        return 1
+
+    python_version = cmaker.CMaker.get_python_version()
+    if (sys.argv[1] == "bin"):
+        print(sys.executable)
+    elif (sys.argv[1] == "lib"):
+        python_lib_path = cmaker.CMaker.get_python_library(python_version).replace("\\", "/")
```

Comment:
You can try something like this SO answer:
https://stackoverflow.com/questions/35071192/how-to-find-out-where-the-python-include-directory-is